### PR TITLE
Tescafé : Refacto selectors

### PR DESCRIPTION
**Pull request by CozyKourai**
- a few data-test-id changes to optimize selector
- New file `selectors.js` containing all selectors

Selectors in the 1st part of the file, in ` // Commons, but with different contexts ` might need some rework, so I added a new card https://trello.com/c/22GedOwK/1850-refacto-selecteurs-am%C3%A9rliorer-les-selecteurs

### CODE REVIEW

**Comment by CozyKourai** [comment]
[(𝐒) Check de tous les selecteurs : Sortir des classes et faire un fichier communs avec tous les selecteurs en exports](https://trello.com/c/lxNhPyAN/1878-%F0%9D%90%92-check-de-tous-les-selecteurs-sortir-des-classes-et-faire-un-fichier-communs-avec-tous-les-selecteurs-en-exports)

File: testcafe/tests/pages/drive/drive-model.js [hunk 1]
```diff
@@ -131,22 +26,23 @@ export default class DrivePage {
   //@param {string} when : text for console.log
   async getContentRowCount(when) {
     //Count only 'real' content row, not the headers
-    const content_rows_count = await this.content_rows.count
+    const contentRows_count = await selectors.contentRows.count
```
Comment (y-lohse) [inline testcafe/tests/pages/drive/drive-model.js]: Nit; but why not `contentRowsCount`?

File: testcafe/tests/pages/photos/photos-timeline-model.js [hunk 2]
```diff
@@ -63,45 +55,45 @@ export default class Timeline extends Commons {
...
-  async waitForLoading() {
-    await t.expect(this.loading.exists).notOk('Page still loading')
-    await isExistingAndVisibile(this.contentWrapper, 'Content Wrapper')
-  }
-
   //@param { number } numOfFiles : number of file to delete
   //@param { bool } isRemoveAll: true if all photos are supposed to be remove at the end
-  async deletePhotos(numOfFiles, isRemoveAll) {
-    await isExistingAndVisibile(this.barPhoto, 'Selection bar')
+  async deltePhotosFromTimeline(numOfFiles, isRemoveAll) {
```
Comment (y-lohse) [inline testcafe/tests/pages/photos/photos-timeline-model.js]: There's a typo in the function name, probably where it's called too.

File: testcafe/tests/drive/classification_scenario.js [hunk 3]
```diff
@@ -213,7 +212,7 @@ test(`${TEST_MOVE_FILE}`, async t => {
     `${FEATURE_PREFIX}/${TEST_MOVE_FILE}-1`,
     true
   )
-  await moveMoadal.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
+  await privateDrivePage.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
```
Comment (y-lohse) [inline testcafe/tests/drive/classification_scenario.js]: Shouldn't this use the second parameter of `moveElementTo`, `inModal`?

File: testcafe/tests/drive/classification_scenario.js [hunk 4]
```diff
@@ -232,7 +231,7 @@ test(`${TEST_MOVE_FOLDER}`, async t => {
     `${FEATURE_PREFIX}/${TEST_MOVE_FOLDER}-1`,
     true
   )
-  await moveMoadal.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
+  await privateDrivePage.moveElementTo(`${FEATURE_PREFIX}-Folder1`)
```
Comment (y-lohse) [inline testcafe/tests/drive/classification_scenario.js]: Same here, should it use the second parameter?

File: testcafe/tests/drive/classification_scenario.js (hunk 3 above)
Comment (CozyKourai) [reply testcafe/tests/drive/classification_scenario.js]: `moveElementTo` doesn't have a second parameter. It call method like `goToFolder` which have a `isModal` parameter.

```
async moveElementTo(destinationFolder) {
    await this.goToBaseFolder(true)
    await this.goToFolder(destinationFolder, true)

    await isExistingAndVisibile(selectors.btnModalSecondButton, 'Move Button')

    await t.click(selectors.btnModalSecondButton)
  }
```

File: testcafe/tests/drive/classification_scenario.js (hunk 4 above)
Comment (CozyKourai) [reply testcafe/tests/drive/classification_scenario.js]: Same answer! :)

**Pull request merged**